examples/combobox/js/combobox-list.js

```diff
@@ -57,7 +57,7 @@ ComboboxList.prototype.init = function () {
 
```

Comment:
There's an error in this file in how the listbox handles the blur event --

line 349 calls `this.removeVisualFocusAll();`, which sets `listbox.hasFocus` to true, which prevents the listbox from closing. The end result is that the listbox does not close when you click outside of it.